src/main/java/com/grocery/net/grocery/model/Order_Info.java

```diff
@@ -0,0 +1,76 @@
+package com.grocery.net.grocery.model;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.Transient;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "Order_Info")
+public class Order_Info {
+
+    @Transient
+    public static final String SEQUENCE_NAME = "users_sequence";
+    @Id
+    private  long order_id;
+    private  String product_count;
+    private double  product_price;
+    private String pay_option;
+    private String product_quantity;
+    private String order_status;
+    private Payment payment;
```

Comment:
We may require order created date. Make use of automatic creation of created date